third_party/proto/ibc/core/commitment/v2/commitment.proto

```
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package ibc.core.commitment.v2;
```

Buf (1.47.2)

Files with package "ibc.core.commitment.v2" must be within a directory "ibc/core/commitment/v2" relative to root but were in directory "third_party/proto/ibc/core/commitment/v2".

(PACKAGE_DIRECTORY_MATCH)